diff --git a/fabric/network.py b/fabric/network.py
index eb8cfb3..630c90b 100644
--- a/fabric/network.py	2012-06-13 16:26:39.000000000 +0100
+++ b/fabric/network.py	2012-06-13 16:26:45.000000000 +0100
@@ -21,7 +21,7 @@
 try:
     import warnings
     warnings.simplefilter('ignore', DeprecationWarning)
-    import ssh
+    import paramiko as ssh
 except ImportError, e:
     import traceback
     traceback.print_exc()
 
diff --git a/setup.py b/setup.py
index af10e2f..5630fd4 100644
--- a/setup.py
+++ b/setup.py
@@ -38,7 +38,6 @@ setup(
     packages=find_packages(),
     test_suite='nose.collector',
     tests_require=['nose', 'fudge<1.0'],
-    install_requires=['ssh>=1.7.14'],
     entry_points={
         'console_scripts': [
             'fab = fabric.main:main',
