--- a/fabric/network.py
+++ b/fabric/network.py
@@ -21,7 +21,9 @@
 try:
     import warnings
     warnings.simplefilter('ignore', DeprecationWarning)
-    import ssh
+    import paramiko as ssh
+    # python-ssh compatibility
+    ssh.io_sleep = 0.01
 except ImportError, e:
     import traceback
     traceback.print_exc()
--- a/setup.py
+++ b/setup.py
@@ -38,7 +38,6 @@
     packages=find_packages(),
     test_suite='nose.collector',
     tests_require=['nose', 'fudge<1.0'],
-    install_requires=['ssh>=1.7.14'],
     entry_points={
         'console_scripts': [
             'fab = fabric.main:main',
