diff --git a/fabric/network.py b/fabric/network.py
index eb8cfb3..630c90b 100644
--- a/fabric/network.py
+++ b/fabric/network.py
@@ -18,7 +18,7 @@ from fabric.utils import abort, handle_prompt_abort
 try:
     import warnings
     warnings.simplefilter('ignore', DeprecationWarning)
-    import ssh
+    import paramiko as ssh
 except ImportError, e:
     print >> sys.stderr, """There was a problem importing our SSH library. Specifically:
 
diff --git a/setup.py b/setup.py
index af10e2f..5630fd4 100644
--- a/setup.py
+++ b/setup.py
@@ -38,7 +38,6 @@ setup(
     packages=find_packages(),
     test_suite='nose.collector',
     tests_require=['nose', 'fudge<1.0'],
-    install_requires=['ssh>=1.7.8'],
     entry_points={
         'console_scripts': [
             'fab = fabric.main:main',
